blocks/editable/format-toolbar/index.js

```diff
@@ -179,7 +179,7 @@ class FormatToolbar extends Component {
 						<UrlInput value={ newLinkValue } onChange={ this.onChangeLinkValue } />
 						<IconButton icon="editor-break" label={ __( 'Apply' ) } type="submit" />
 						<IconButton icon="editor-unlink" label={ __( 'Remove link' ) } onClick={ this.dropLink } />
-						<IconButton icon="admin-generic" onClick={ this.toggleLinkSettingsVisibility } aria-expanded={ settingsVisible } />
+						<IconButton icon="admin-generic" label={ __( 'Link Settings' ) } onClick={ this.toggleLinkSettingsVisibility } aria-expanded={ settingsVisible } />
```

Comment:
This is a long line, I'd suggest we split props onto multiple lines to improve readability.

Reply:
Done!